File: Application/Services/Operations/Main/Customers/Dtos/CustomerDto.cs
using System;
using Application.Services.Operations.Main.Inheritances;
using Domain.Entities.Main.Customers;

namespace Application.Services.Operations.Main.Customers.Dtos
{
    public class CustomerDto : MainEntitiesBaseDto
    {
        public DateTime Assured { get; set; }
        public decimal Payment { get; set; }
        public DateTime Expires { get; set; }
        public decimal Discount { get; set; }
        public AdditionalCostsDto AdditionalCosts { get; set; }
    }
}

File: Views/src/components/main/customer/components/commons-components/dtos/customer-dto.ts

import { MainEntitiesBase } from '../../../../../../components/main/inheritances/dtos/main-entities-base';
import { AdditionalCosts } from './additional-costs';

export class CustomerDto extends MainEntitiesBase {
  constructor() {
    super()
  }

  assured: boolean;
  payment: number;
  expiration: number;
  disabled: boolean;
  discount: number;
  additionalCosts: AdditionalCosts;

  //  servicesExecuted:BudgetServiceDto[];
  // electronicsRepairs:ElectronicRepairDto[];
}

